vscode/src/completions/providers/fireworks.ts

```diff
@@ -26,9 +26,11 @@ import {
     tracer,
 } from '@sourcegraph/cody-shared'
 import { fetch } from '@sourcegraph/cody-shared'
-import { getLanguageConfig } from '../../tree-sitter/language'
+import { type LanguageConfig, getLanguageConfig } from '../../tree-sitter/language'
 import { getSuffixAfterFirstNewline } from '../text-processing'
 import { forkSignal, generatorWithTimeout, zipGenerators } from '../utils'
+import * as fim_prompt_utils from './fim-prompt-utils'
```

Comment:
I like snake case more for readability, but in the TS world, we use camel case for everything. Let's use it for consistency.
```suggestion
import * as fimPromptUtils from './fim-prompt-utils'
```

Reply:
I also felt a bit uneasy while making changes -- I realize now this was due to this variable name 😅
I mixed python, ts -- fixed this, thanks for pointing it out !!